### PR TITLE
Solved the issue with Pan y-axis inverted in frameBuffer.

### LINKED ISSUE #6538: Pan y-axis inverted in frameBuffer
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [X] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

1.8.0

### Web browser and version

Any

### Operating System

Any

### Steps:
1. Use framebuffer with orbitControl inside
2. Drag clicking right mouse button
3.

### Snippet:

```js
let layer;
function setup() {
  createCanvas(400, 400, WEBGL);
  layer = createFramebuffer();
}

function draw() {
  layer.begin();
  orbitControl();
  background(220);
  sphere(150);
  layer.end();
  image(layer,-width/2,-height/2)
}

```

### PR DESCRIPTION
**Pull request by perminder-17**
Resolves #6538 

 Changes:

https://editor.p5js.org/aman12345/sketches/Fu-bC-xWm

### CODE REVIEW

**Comment by perminder-17** [comment]
I am not entirely sure if this was supposed to be done here. @davepagurek

**Comment by davepagurek** [comment]
I believe now it's inverted if you *aren't* using a framebuffer:
```js
let layer;
function setup() {
  createCanvas(400, 400, WEBGL);
  layer = createFramebuffer();
}

function draw() {
  // layer.begin();
  orbitControl();
  background(220);
  sphere(150);
  // layer.end();
  // image(layer,-width/2,-height/2)
}
```

I think rather than multiplying by -1, we should be multiplying by the `yScale` property of the active camera, which will be -1 for framebuffers but 1 for the main canvas camera.

File: src/webgl/interaction.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ p5.prototype.orbitControl = function(
         deltaPhi = sensitivityY * this.movedY / scaleFactor;
       } else if (this.mouseButton === this.RIGHT) {
         moveDeltaX = this.movedX;
-        moveDeltaY = (this.movedY * -1);
+        moveDeltaY = (this.movedY * (cam.projMatrix.mat4[5] < 0 ? 1 : -1));
```
Comment (davepagurek) [inline src/webgl/interaction.js:232]: Is there a reason for accessing the projection matrix rather than using `cam.yScale`?

Comment (perminder-17) [reply src/webgl/interaction.js]: Sorry, i just confused with the y-scale. now i have done the changes. it's working fine as you told

Comment (perminder-17) [reply src/webgl/interaction.js]: I guess both have the same sign?

**Review by davepagurek — COMMENTED** [review]
I think this works, although we probably could remove the brackets for legibility. Just to confirm, is your sketch updated with the latest version of the code? The sketch seems to work well

**Comment by perminder-17** [comment]
> I think this works, although we probably could remove the brackets for legibility. Just to confirm, is your sketch updated with the latest version of the code? The sketch seems to work well

yes..my code is up to date

**Review by davepagurek — APPROVED** [review]
Looks good then, thanks!

**Comment by perminder-17** [comment]
Thank you for catching mistakes. I decided to use a projection matrix because it was working, and I just thought if yscaling cause any issues further.

File: src/webgl/interaction.js (hunk 1 above)
Comment (davepagurek) [reply src/webgl/interaction.js]: Right, I think the yScale value is used in creating the projection matrices. In the end I think they should be equivalent, but it's maybe just more readable to see a value like `yScale` than an index into a matrix.

**Pull request merged**